src/app/stepprocess/stepprocess-auto-step-change.demo.html

```diff
@@ -8,21 +8,21 @@
 
     <ng-template ngFor let-step [ngForOf]="steps">
 
-      <li soho-step-list-item *ngIf="step.content">
+      <li soho-step-list-item stepId="{{step.id}}" *ngIf="step.content">
```

Comment:
why go adding a new input when you could just do something like:
`[isSelected]={{step.id == selectedStepId}}`